io.github.TheWisker.Cavasik.json

```diff
@@ -0,0 +1,83 @@
+{
+    "app-id" : "io.github.TheWisker.Cavasik.desktop",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "44",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "cavasik",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--socket=pulseaudio",
+        "--device=dri"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+    	"shared-modules/linux-audio/fftw3f.json",
+    	{
+    	    "name" : "iniparser",
+    	    "buildsystem" : "simple",
+    	    "build-commands" :
+    	    	[
+    	    	    "make PREFIX=/app",
+    	    	    "install -Dm0644 src/iniparser.h /app/include/iniparser.h",
+    	    	    "install -Dm0644 src/dictionary.h /app/include/dictionary.h",
+    	    	    "install -Dm0644 libiniparser.so.1 /app/lib/libiniparser.so.1",
+    	    	    "ln -sf libiniparser.so.1 /app/lib/libiniparser.so"
+    	    	],
+    	    "sources" : [
+    	        {
+    	            "type" : "git",
+    	            "url" : "https://github.com/ndevilla/iniparser.git",
+    	            "commit" : "deb85ad4936d4ca32cc2260ce43323d47936410d"
+    	        }
+    	    ]
+    	},
+    	{
+    	    "name" : "cava",
+    	    "sources" : [
+    	    	{
+    	    	    "type" : "git",
+    	    	    "url" : "https://github.com/karlstav/cava.git",
+    	    	    "tag" : "0.8.3",
+    	    	    "commit" : "746a3b1e6021e383aea9d0000f49d71fb24e1856"
+    	    	}
+    	    ]
+    	},
+    	{
+	    "name": "python3-pydbus",
+	    "buildsystem": "simple",
+	    "build-commands": [
+		"pip3 install --verbose --exists-action=i --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} \"pydbus\" --no-build-isolation"
+	    ],
+	    "sources": [
+		{
+		    "type": "file",
+		    "url": "https://files.pythonhosted.org/packages/92/56/27148014c2f85ce70332f18612f921f682395c7d4e91ec103783be4fce00/pydbus-0.6.0-py2.py3-none-any.whl",
+		    "sha256": "66b80106352a718d80d6c681dc2a82588048e30b75aab933e4020eb0660bf85e"
+		}
+	    ]
+	},
+        {
+            "name" : "cavasik",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "dir",
+                    "path" : "."
```

Comment:
That will not work. You have to download your source from the Internet like the other modules.